app/controllers/submissions_controller.rb

```diff
@@ -0,0 +1,77 @@
+class SubmissionsController < ApplicationController
+  before_action :authenticate_user!, only: [:create, :create_comment]
+  before_action :authenticate_to_submit, only: :new
+
+  helper_method :posts, :post, :new_post, :new_comment
```

Comment:
Oooh neat. I didn't know this was a thing.

Reply:
Do you mean listing the helper_methods at the top after the before filters? I thought they'd be easier to quickly see there.... though some people may look for `helper_method :posts` right after the private method.

Reply:
Yep, `:helper_method`. I've never seen it used before.

Reply:
And I think it makes sense up here, where it's really easy to catch when scanning.